assets/src/edit-story/components/checklist/checks/storyPosterAttached.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+// import { __ } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app/story';
+import { hasNoFeaturedMedia } from '../utils';
+// import ChecklistCard from '../../../checklistCard';
+
+export function storyHasNoPosterAttached(story) {
+  return (
+    typeof story.featuredMedia?.url !== 'string' || hasNoFeaturedMedia(story)
+  );
+}
+
+export function StoryPosterAttached() {
+  //@TODO refine this context selector and storyHasNoPosterAttached to run more selectively
+  const story = useStory(({ state }) => state);
+  return storyHasNoPosterAttached(story)
+    ? // <ChecklistCard
+      //   title={__('Add Web Story poster image', 'web-stories')}
+      //   titleProps={{
+      //     onClick: () => {
+      //       /* perform highlight here */
+      //     },
+      //   }}
+      //   footer={
+      //     <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
+      //       <ul>
+      //         <li>
+      //           {__('Use as a representation of the story.', 'web-stories')}
+      //         </li>
+      //         <li>{__('Avoid images with embedded text.', 'web-stories')}</li>
+      //         <li>
+      //           {__("Use an image that's at least 640x853px.", 'web-stories')}
+      //         </li>
+      //         <li>{__('Maintain a 3:4 aspect ratio.', 'web-stories')}</li>
+      //       </ul>
```

Comment:
💭 Should we use the [`<List>` component](https://github.com/google/web-stories-wp/pull/7967/files#diff-24a77714d0a9e37f95e8bc53b8920524a2cdcf3e85dda9cd183f7c2672eed41cR29) here? Not sure if those styles are necessary

Reply:
we will. Gonna do a separate PR to actually use the new card components, just wanted to set a rough blueprint here with the commented out code